packages/cli/src/cli/config/action.import.ts

```diff
@@ -176,7 +176,7 @@ export class CommandImport extends CommandLineAction {
           let msg = ' - Zoom level updated.';
           if (layer.minZoom !== existing.minZoom) msg += ` min zoom ${existing.minZoom} -> ${layer.minZoom}`;
           if (layer.maxZoom !== existing.maxZoom) msg += ` max zoom ${existing.maxZoom} -> ${layer.maxZoom}`;
-          change.push(`${msg}\n`);
+          change.push(`${msg} -- [Aerial]((${PublicUrlBase}?config=${this.config.value}&debug))\n`);
```

Comment:
I think this url should be added to the next if statements? we are going to need both NZTM and Webmercator links?

What happens if the zoom changes and the layer id changes? going to get a bunch of links?

Reply:
Good point, we shouldn't end up with too many links. I have update this only have a default aerial preview link if only zoom level change and layer id not change.

Reply:
I add the links for both available projections now.

![image](https://user-images.githubusercontent.com/12163920/220232232-3e1e73e2-2958-4dda-a0cc-e3b5a4ef9c4d.png)